ui/components/ui/error-message/index.scss

```
@@ -2,14 +2,14 @@
   @include H7;
 
   min-height: 32px;
-  border: 1px solid $monzo;
-  color: $monzo;
-  background: lighten($monzo, 56%);
-  border-radius: 4px;
+  border: 1px solid $Red-300;
+  color: $Black-100;
+  background: $Red-000;
```

Ideally we wouldn't use $Red-300 but instead one of our $error-1/2/3 variants. 
```suggestion
  border: 1px solid $Red-300;
  color: $ui-black;
  background: $error-2;
```

---

but also i'm not gonna block on it.